tests/test_rest_consumer.py

```diff
@@ -308,41 +288,28 @@ async def test_publish_consume_avro(kafka_rest, karapace, aiohttp_client, admin_
         assert expected == actual, f"Expecting {actual} to be {expected}"
 
 
-async def test_remote():
-    def mock_factory(app_name):
-        def inner():
-            app = MagicMock()
-            app.type = app_name
-            app.serializer = MagicMock()
-            app.consumer_manager = MagicMock()
-            app.serializer.registry_client = MagicMock()
-            app.consumer_manager.deserializer = MagicMock()
-            app.consumer_manager.hostname = "http://localhost:8082"
-            app.consumer_manager.deserializer.registry_client = MagicMock()
-            return app, None
-
-        return inner
-
-    if "REST_URI" not in os.environ or "REGISTRY_URI" not in os.environ:
-        pytest.skip("Cannot run remote test: REST_URI and REGISTRY_URI env vars set")
-    # from here on we assume they all work on the same host
-    print("Test publish consume avro")  # OK
-    await test_publish_consume_avro(mock_factory("rest"), mock_factory("registry"), None, KafkaRestAdminClient())
-    print("Test consume")  # OK
-    await test_consume(mock_factory("rest"), None, KafkaRestAdminClient(), KafkaProducer())
-    print("Test offsets")  # OK
-    await test_offsets(mock_factory("rest"), None, KafkaRestAdminClient())
-    print("Test seek")  # OK
-    await test_seek(mock_factory("rest"), None, KafkaRestAdminClient())
-    print("Test subscription")  # OK if we do another consume and force a sub recheck
-    await test_subscription(mock_factory("rest"), None, KafkaRestAdminClient(), KafkaProducer())
-    print("Test assignment")  # OK
-    await test_assignment(mock_factory("rest"), None, KafkaRestAdminClient())
-    print("Test create delete")  # OK
-    await test_create_and_delete(mock_factory("rest"), None)
-    # # why not
-    import tests.test_rest as rs
-    print("Test avro publish")  # OK , need to look at compatibility checks though
-    await rs.test_avro_publish(mock_factory("rest"), mock_factory("registry"), None, KafkaRestAdminClient())
-    print("Test what's left i guess")  #
-    await rs.test_local(mock_factory("rest"), None, KafkaProducer(), KafkaRestAdminClient())
+#
+# async def test_remote():
```

Comment:
Commented out? If not needed please remove it alltogether.

Reply:
done